cloud_launch.json

```diff
@@ -36,15 +38,22 @@
         }
       ]
     },
+    {
+      "worker_type": "UnityLoadBalancer",
+      "permissions": [
+        {
+          "all": {}
+        }
+      ]
+    },
+
     {
       "worker_type": "UnityClient",
       "permissions": [
         {
           "entity_query": {
             "allow": true,
-            "components": [
-              "*"
-            ]
+            "components": ["*"]
```

Comment:
Do we still want to fix this by having the PlayerCreator hardcode it's entity id?

Reply:
_sigh_